service_provider_config.go

```diff
@@ -53,6 +55,43 @@ const (
 	AuthenticationTypeHTTPDigest AuthenticationType = "httpdigest"
 )
 
+// BulkSchema specifies a supported bulk endpoint spec.
+type BulkSchema struct {
+	support        bool
+	maxOperations  int
+	maxPayloadSize int
+}
+
+// These numbers are base on https://tools.ietf.org/html/rfc7644#section-3.12
+const (
+	defaultMaxOperations  = 1000
+	defaultMaxPayloadSize = 1048576 // 1 MB
+)
+
+// SupportBulkSchema specifies service provider can support /Bulk endpoint.
+func SupportBulkSchema(maxOperations, maxPayloadSize int) BulkSchema {
+	if maxOperations > defaultMaxOperations {
+		maxOperations = defaultMaxOperations
+	}
+	if maxPayloadSize > defaultMaxPayloadSize {
+		maxOperations = defaultMaxPayloadSize
+	}
+	return BulkSchema{
+		support:        true,
+		maxOperations:  maxOperations,
+		maxPayloadSize: maxPayloadSize,
+	}
+}
+
+// UnsupportBulkSchema specifies service provider can NOT support /Bulk endpoint.
+func UnsupportBulkSchema() BulkSchema {
+	return BulkSchema{
+		support:        false,
+		maxOperations:  defaultMaxOperations,
+		maxPayloadSize: defaultMaxPayloadSize,
+	}
+}
```

Comment:
Boolean values will be false by default, no need to declare it explicitly.

For the `maxOperations` and `maxPayloadSize` I would take a look at:
https://github.com/elimity-com/scim/blob/307c22b7fcf7291d390e867f456e5b87f08f2447/service_provider_config.go#L85-L91